test/api_test.py

```diff
@@ -19,11 +19,18 @@
 import logging
 import unittest
 
+import googleapiclient.errors
+import mock
+import nose.tools
+
 import hyou.api
 
 import http_mocks
```

Comment:
I don't feel strongly about it, just left it the way it was 😁  But I'll change it to the "3 groups" format in the test modules.